corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -208,4 +226,45 @@ <h4 class="modal-title">{% blocktrans with couch_user.human_friendly_name as fri
       </div>
     </div>
   {% endif %}
+
+  <div id="force_user_412_{{ couch_user.user_id }}" class="modal fade">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title">{% blocktrans with couch_user.human_friendly_name as friendly_name %}Hard Refresh Device Data for Mobile Worker {{ friendly_name }}?{% endblocktrans %}
+          </h4>
+        </div>
+        <form class="form-horizontal"
+              style="margin: 0; padding: 0"
+              action="{% url "force_user_412" domain couch_user.user_id %}"
+              method="post"
+              data-bind="submit: submit">
```

Comment:
What's this doing? I don't see knockout bindings being applied to this content.

Reply:
You caught me copy-pasting! Thanks for pointing it out, I'll clean up this and other similar places that have copied irrelevant code.